api/test/e2enew/server_info/server_info_test.go

```diff
@@ -14,127 +14,113 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package e2e
+package server_info
 
 import (
 	"net/http"
-	"testing"
-	"time"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+
+	"e2enew/base"
 )
 
-func TestServerInfo_Get(t *testing.T) {
-	// wait for apisix report
-	time.Sleep(2 * time.Second)
-	testCases := []HttpTestCase{
-		{
-			Desc:         "get server info",
-			Object:       ManagerApiExpect(t),
+var _ = ginkgo.Describe("server info test", func() {
+	table.DescribeTable("get server info",
+		func(tc base.HttpTestCase) {
+			base.RunTestCase(tc)
+		},
+		table.Entry("get server info(apisix-server1)", base.HttpTestCase{
+			Object:       base.ManagerApiExpect(),
 			Path:         "/apisix/admin/server_info/apisix-server1",
 			Method:       http.MethodGet,
-			Headers:      map[string]string{"Authorization": token},
+			Headers:      map[string]string{"Authorization": base.GetToken()},
 			ExpectStatus: http.StatusOK,
 			ExpectBody:   "\"hostname\":\"apisix_server1\"",
-		},
-		{
-			Desc:         "get server info",
-			Object:       ManagerApiExpect(t),
+			Sleep:        base.SleepTime * 15,
```

Comment:
why sleep so long ..

Reply:
Old code wait for APISIX report, sleep  2 seconds... So I sleep a little longer
```
	// wait for apisix report
	time.Sleep(2 * time.Second)
```

Reply:
@Jaycean 
It is recommended to directly use the original time - 2 seconds.

Some places may rely on services that are delayed. For example, here is the reporting interval of the plugin `server-info`.

Reply:
done.